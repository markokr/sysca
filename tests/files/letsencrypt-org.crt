-----BEGIN CERTIFICATE-----
MIIHMzCCBhugAwIBAgISBCCadDETtK0fBJclSNL5VM/4MA0GCSqGSIb3DQEBCwUA
MEoxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MSMwIQYDVQQD
ExpMZXQncyBFbmNyeXB0IEF1dGhvcml0eSBYMzAeFw0xOTA3MTUyMjA2MzhaFw0x
OTEwMTMyMjA2MzhaMB4xHDAaBgNVBAMTE3d3dy5sZXRzZW5jcnlwdC5vcmcwggEi
MA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQC+/qreemgMT56XozaPpE3GjSI6
ElM/6pQ+CcyXD0mFjzLyuuDZrDxPMntOfrl5bYD/HD2nAX0yq1e/MC5MHtG5IK+I
2dvbKq5wcK87TzNU+jETOGOf4cIzPPD/+hZXy0d9wf0nATq/7Ezm2MVdhsGCrjW7
46WiGrhzYPVQBaznTdsXpn9npmiqUpoDC/QceKg5ZMslgDHTVpK/HgHsNNDNk35Z
KGqbupRtSmcIZiIzz41pabdyiV/z0l4CMQAAaxy9SI2eWyW5MDp0IHh7qpj7t0cj
G8Tnp7TYCpEt5NKwwEbVyg0xy6X5Ezj60sneqrkmNHnQlaFpg+kEgsArfpj1AgMB
AAGjggQ9MIIEOTAOBgNVHQ8BAf8EBAMCBaAwHQYDVR0lBBYwFAYIKwYBBQUHAwEG
CCsGAQUFBwMCMAwGA1UdEwEB/wQCMAAwHQYDVR0OBBYEFHStWB8/pVkpShYlQpw4
Acg3XWgHMB8GA1UdIwQYMBaAFKhKamMEfd265tE5t6ZFZe/zqOyhMG8GCCsGAQUF
BwEBBGMwYTAuBggrBgEFBQcwAYYiaHR0cDovL29jc3AuaW50LXgzLmxldHNlbmNy
eXB0Lm9yZzAvBggrBgEFBQcwAoYjaHR0cDovL2NlcnQuaW50LXgzLmxldHNlbmNy
eXB0Lm9yZy8wggHxBgNVHREEggHoMIIB5IIbY2VydC5pbnQteDEubGV0c2VuY3J5
cHQub3JnghtjZXJ0LmludC14Mi5sZXRzZW5jcnlwdC5vcmeCG2NlcnQuaW50LXgz
LmxldHNlbmNyeXB0Lm9yZ4IbY2VydC5pbnQteDQubGV0c2VuY3J5cHQub3Jnghxj
ZXJ0LnJvb3QteDEubGV0c2VuY3J5cHQub3Jngh9jZXJ0LnN0YWdpbmcteDEubGV0
c2VuY3J5cHQub3Jngh9jZXJ0LnN0Zy1pbnQteDEubGV0c2VuY3J5cHQub3JngiBj
ZXJ0LnN0Zy1yb290LXgxLmxldHNlbmNyeXB0Lm9yZ4ISY3AubGV0c2VuY3J5cHQu
b3JnghpjcC5yb290LXgxLmxldHNlbmNyeXB0Lm9yZ4ITY3BzLmxldHNlbmNyeXB0
Lm9yZ4IbY3BzLnJvb3QteDEubGV0c2VuY3J5cHQub3Jnghtjcmwucm9vdC14MS5s
ZXRzZW5jcnlwdC5vcmeCD2xldHNlbmNyeXB0Lm9yZ4IWb3JpZ2luLmxldHNlbmNy
eXB0Lm9yZ4IXb3JpZ2luMi5sZXRzZW5jcnlwdC5vcmeCFnN0YXR1cy5sZXRzZW5j
cnlwdC5vcmeCE3d3dy5sZXRzZW5jcnlwdC5vcmcwTAYDVR0gBEUwQzAIBgZngQwB
AgEwNwYLKwYBBAGC3xMBAQEwKDAmBggrBgEFBQcCARYaaHR0cDovL2Nwcy5sZXRz
ZW5jcnlwdC5vcmcwggEEBgorBgEEAdZ5AgQCBIH1BIHyAPAAdwB0ftqDMa0zEJEh
nM4lT0Jwwr/9XkIgCMY3NXnmEHvMVgAAAWv341OhAAAEAwBIMEYCIQD+YjTpDPY/
XE583hnJ/P7nGYeJwaiXPaBXN8QC6/skIQIhAP3iIJpxlSuLM2O4z8DV3M/DMeGk
hN7G00c+4wsYVp2BAHUAKTxRllTIOWW6qlD8WAfUt2+/WHopctykwwz05UVH9HgA
AAFr9+NTgwAABAMARjBEAiAY3SVnRrFqpscOQ3vmhr98G5JDcxOj4SvU2b8PuFMa
CgIgO7kNG7OQ4hXDoaaZrSCPje/8IziAyYLNqOB5sNoC1g0wDQYJKoZIhvcNAQEL
BQADggEBAAyAvs+BMiz/qGHn2selRXdSvla7X+catL3SQDH5xhOSvOa46nX8WbKx
c2nTYiQll54sWtpb5YNGLhweYdNmCl+7oafuflsPmi+yA4Zeb48piLI9fr2kKz6W
CKJx54znvwRYQR6i2pz90YWJlLpge5RzRqjIsYKujtLsVrR7V8+JXMldcUka5lwk
58wnucBJnPSZa9/wyaEKhcjMOAKf1EbTCrMXp+Xl9eQhzXkbqbr1GH5fSYWwCgoa
IZoizDESs/lXSAav6orOrVAaQd6VSka5Hvm808LI3pocqsNmFyNAxh5I6Fc74hPl
+JF8ZrzXShHJyMomroLcXo1kKZxBBGk=
-----END CERTIFICATE-----
