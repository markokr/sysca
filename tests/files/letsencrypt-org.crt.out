Version: 3
Public key: rsa:2048
Not Valid Before: 2019-07-15 22:06:38
Not Valid After: 2019-10-13 22:06:38
Serial: 04:20:9a:74:31:13:b4:ad:1f:04:97:25:48:d2:f9:54:cf:f8
Subject: CN = www.letsencrypt.org
Subject Alternative Name:
  dns:cert.int-x1.letsencrypt.org
  dns:cert.int-x2.letsencrypt.org
  dns:cert.int-x3.letsencrypt.org
  dns:cert.int-x4.letsencrypt.org
  dns:cert.root-x1.letsencrypt.org
  dns:cert.staging-x1.letsencrypt.org
  dns:cert.stg-int-x1.letsencrypt.org
  dns:cert.stg-root-x1.letsencrypt.org
  dns:cp.letsencrypt.org
  dns:cp.root-x1.letsencrypt.org
  dns:cps.letsencrypt.org
  dns:cps.root-x1.letsencrypt.org
  dns:crl.root-x1.letsencrypt.org
  dns:letsencrypt.org
  dns:origin.letsencrypt.org
  dns:origin2.letsencrypt.org
  dns:status.letsencrypt.org
  dns:www.letsencrypt.org
Usage: digital_signature, key_encipherment, server, client
Subject Key Identifier: 74ad581f3fa559294a1625429c3801c8375d6807
Authority Key Identifier: a84a6a63047dddbae6d139b7a64565eff3a8eca1
Issuer Name: C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
Issuer URLs: http://cert.int-x3.letsencrypt.org/
OCSP URLs: http://ocsp.int-x3.letsencrypt.org
Certificate Policies:
  2.23.140.1.2.1
  1.3.6.1.4.1.44947.1.1.1:|P=http://cps.letsencrypt.org|
Precert Signed Timestamps:
  v1_PRE_CERTIFICATE - 2019-07-15 23:06:38.881000 - 747eda8331ad331091219cce254f4270c2bffd5e422008c6373579e6107bcc56
  v1_PRE_CERTIFICATE - 2019-07-15 23:06:38.851000 - 293c519654c83965baaa50fc5807d4b76fbf587a2972dca4c30cf4e54547f478
